internal/distro/fedora/distro.go

```diff
@@ -319,6 +319,24 @@ var (
 		payloadPipelines: []string{"os", "container"},
 		exports:          []string{"container"},
 	}
+
+	minimalrpmImgType = imageType{
+		name:     "minimal-raw",
+		filename: "minimal-raw.img",
```

Comment:
Usually the filenames we specify here aren't unique or specific to an image type, but simple and direct descriptions of the artifact (e.g., `installer.iso`, `container.tar`, `commit.tar`, `disk.raw`, `disk.qcow2`).
I'd change this to `raw.img` or `image.raw`.

This isn't important, just a matter of consistency.